streaming_providers/seedr/utils.py

```
@@ -1,242 +1,261 @@
-    return seedr
+async def wait_for_completion(
+    seedr: Seedr, info_hash: str, max_retries: int = 1, retry_interval: int = 1
+) -> None:
+    """Wait for torrent to complete downloading."""
+    for _ in range(max_retries):
+        status, data = await check_torrent_status(seedr, info_hash)
+
+        if status == TorrentStatus.COMPLETED:
+            return
+        elif status == TorrentStatus.DOWNLOADING and data.get("progress") == "100":
+            return
```

Ruff

Combine `if` branches using logical `or` operator

Combine `if` branches

(SIM114)